### ISSUE TITLE
Demo looks different from the flask app

### ISSUE THREAD

**Issue by Clara-liu** [aadityaubhat/auto-analyst#5]
Thank you for addressing my previous issue. I managed to spin up the app with the new instructions. However, there is very little documentation on usage and the demo looks quite different from the current version of the app (e.g. no import data file utility).

May I ask if this is still a work in progress (not able to run like the demo) or is there more documentation coming soon to address usage? Many thanks!

**Comment by aadityaubhat** [comment]
Hi Clara, I have updated the readme to reflect the new UI. Let me know if it is useful. The project is meant to be a self service platform which interacts with the databases, hence I removed the csv file upload functionality. But if you are interested in interacting with a CSV, I can add that as a database connection.

If you let me know your exact use-case, I can definitely prioritize the features to support it.
